Emby.Server.Implementations/Channels/ChannelManager.cs

```diff
@@ -1156,7 +1156,7 @@ private async Task<BaseItem> GetChannelItemEntityAsync(ChannelItemInfo info, ICh
 
                 if (info.People is not null && info.People.Count > 0)
                 {
-                    _libraryManager.UpdatePeople(item, info.People);
+                    await _libraryManager.UpdatePeopleAsync(item, info.People, cancellationToken).ConfigureAwait(false);
```

Comment:
Btw, do we have a synchronization context somewhere?
I mean, do we really need `ConfigureAwait` everywhere?

Reply:
No, but we should keep using them for the stuff that gets published to nuget at least